pkg/csi/cinder/openstack/openstack_mock.go

```diff
@@ -217,22 +217,22 @@ func (_m *OpenStackMock) ListSnapshots(limit int, offset int, filters map[string
 	return r0, r1
 }
 
-// CreateSnapshot provides a mock function with given fields: name, volID, description, tags
-func (_m *OpenStackMock) CreateSnapshot(name string, volID string, description string, tags *map[string]string) (*snapshots.Snapshot, error) {
-	ret := _m.Called(name, volID, description, tags)
+// CreateSnapshot provides a mock function with given fields: name, volID, tags
+func (_m *OpenStackMock) CreateSnapshot(name string, volID string, tags *map[string]string) (*snapshots.Snapshot, error) {
```

Comment:
you need to make changes here also
https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/sanity/fakecloud.go#L131